frontend/app/src/components/asset-manager/newly-detected/NewlyDetectedAssetTable.vue

```diff
@@ -0,0 +1,249 @@
+<script setup lang="ts">
+import { type ComputedRef, type Ref } from 'vue';
+import { Blockchain } from '@rotki/common/lib/blockchain';
+import { type NewDetectedToken } from '@/types/websocket-messages';
+import type {
+  DataTableColumn,
+  DataTableSortColumn
+} from '@rotki/ui-library-compat/dist/components';
```

Comment:
Don't we export them from elsewhere?

Reply:
auto imports, should be exported via `@rotki/ui-library-compat`